static/js/components/StaffLearnerInfoCard.js

```diff
@@ -100,8 +102,20 @@ const formatCourseGrade = R.compose(
 );
 
 const StaffLearnerInfoCard = (props: StaffLearnerCardProps) => {
-  const { program, dialogVisibility, setShowGradeDetailDialog } = props;
+  const { program, dialogVisibility, setShowGradeDetailDialog, prices } = props;
   const { totalPassedCourses, totalCourses } = programCourseInfo(program);
+  let calculatedCoursePrice = 0;
+
+  for (const course of program.courses) {
+    for (const run of course.runs) {
+      let coursePrice = prices.pricesInclCouponByRun.get(run.id);
+      if (coursePrice) {
+        calculatedCoursePrice = R.add(calculatedCoursePrice, coursePrice.price);
```

Comment:
I'm not sure about that, maybe @noisecapella would know if that's what we want to use?